.github/workflows/main.yml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - "main"
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Run tests every day at midnight to make sure no dependency updates break the library